tools/lerno/README.md

```diff
@@ -0,0 +1,37 @@
+lerno
```

Comment:
Let's make this a public tool from day 1. Otherwise, it will never be open sourced and will slowly take too much assumptions related to our repo.